src/data/sym/basic.lean

```diff
@@ -177,7 +209,7 @@ lemma exists_eq_cons_of_succ (s : sym α n.succ) : ∃ (a : α) (s' : sym α n),
 begin
   obtain ⟨a, ha⟩ := exists_mem s,
   classical,
-  exact ⟨a, s.erase a ha, (s.cons_erase _ _).symm⟩,
```

Comment:
Do you need the `@` here?

Reply:
Apparently not, a simple `rw cons_erase` is sufficient